frontend/.env.local.example

```
@@ -0,0 +1,10 @@
+# Frontend Environment Configuration for Local Development
+# Copy this file to frontend/.env.local and update values for your environment
+
+# API Server URL
+NEXT_PUBLIC_API_URL=http://localhost:8080
+
+# Clerk Authentication (Optional for local development)
+# Leave empty to run in dev mode without authentication
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=
+CLERK_SECRET_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CLERK_SECRET_KEY key should go before the NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY key

(UnorderedKey)